bedrock/bedrock-common/src/main/java/com/nukkitx/protocol/bedrock/packet/SetSpawnPositionPacket.java

```diff
@@ -10,8 +10,25 @@
 @Data
 @EqualsAndHashCode(doNotUseGetters = true, callSuper = false)
 public class SetSpawnPositionPacket extends BedrockPacket {
+    // spawnType is the type of spawn to set. It is either PLAYER_SPAWN or WORLD_SPAWN, and specifies
+    // the behaviour of the spawn set. If WORLD_SPAWN is set, the position to which compasses will point is
+    // also changed.
```

Comment:
Could you use proper Javadoc annotations here?

Reply:
Can you give me an example? As far as I know there is no such thing as javadoc for class variables unless you run this through delombok and run javadoc over that?